static/app/views/detectors/components/detectorListTable.tsx

```diff
@@ -40,13 +40,13 @@ function DetectorListTable({detectors}: DetectorListTableProps) {
           <HeaderDivider />
           <Heading>{t('Type')}</Heading>
         </Flex>
-        <Flex className="last-issue">
+        <Flex className="issue">
           <HeaderDivider />
           <Heading>{t('Last Issue')}</Heading>
         </Flex>
-        <Flex className="owner">
+        <Flex className="creator">
           <HeaderDivider />
-          <Heading>{t('Owner')}</Heading>
+          <Heading>{t('Creator')}</Heading>
```

Comment:
The Figma design [here](https://www.figma.com/design/99EtM4v3UYfGCN0QJU1QfI/Alerts-are-Issues?node-id=1201-17956&t=cliXT6hLM7dXTJ3G-0) says Owner, unless I'm looking at the wrong place?

Reply:
Good catch! Fixed in 458155f4234